anydex/test/wallet/stellar/test_stellar_wallet.py

```diff
@@ -0,0 +1,308 @@
+import abc
```

Comment:
Due to consistency with wallet class names and _test_xlm_provider.py_, file should probably be named _test_xlm_wallet.py_